app/controllers/jobs_controller.rb

```diff
@@ -3,52 +3,67 @@
 class JobsController < ApplicationController
   before_action :authenticate_user!
 
-  before_action :set_job, only: [:show, :edit, :update, :destroy, :retry]
+  before_action :set_job, only: [:show, :edit, :update, :destroy, :retry, :inform]
 
   respond_to :html
 
+  # GET /jobs
   def index
     @jobs = Job.order('created_at DESC').page(params[:page]).per(params[:per_page])
     @jobs = @jobs.where(application_id: params[:app_id]) if params[:app_id]
     @jobs = @jobs.incomplete if params[:status] == 'incomplete'
     @jobs = @jobs.failed if params[:status] == 'failed'
-
-    respond_with(@jobs)
+    @jobs
   end
 
+  # GET /jobs/1
   def show
-    respond_with(@job)
   end
 
+  # GET /jobs/new
   def new
     @job = Job.new
-    respond_with(@job)
   end
 
+  # GET /jobs/1/edit
   def edit
   end
 
+  # POST /jobs
   def create
     @job = Job.new(job_params)
-    @job.save
-    respond_with(@job)
+    if @job.save
+      redirect_to @job, notice: 'Job was successfully created.'
+    else
+      render :new
+    end
   end
 
+  # PATCH/PUT /jobs/1
   def update
-    @job.update(job_params)
-    respond_with(@job)
+    if @job.update(job_params)
+      redirect_to @job, notice: 'Job was successfully updated.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.